pylib/tools/genhooks.py

```diff
@@ -67,6 +67,16 @@
         Your add-on can check filter_name to decide whether it should modify
         field_text or not before returning it.""",
     ),
+    Hook(
+        name="note_is_being_flushed",
+        args=["note: Note"],
+        doc="Allow to change a note before adding it to the database.",
```

Comment:
"before it is added/updated in the database"

Reply:
Right. Corrected